src/test/java/uk/gov/hmcts/reform/managecase/security/SecurityUtilsTest.java

```diff
@@ -38,30 +39,30 @@ public void setup() {
 
     @Test
     void hasSolicitorRoleReturnsFalseWhenRolesEmpty() {
-        assertFalse(securityUtils.hasSolicitorRole(roles));
+        assertFalse(securityUtils.hasSolicitorRoleForJurisdiction(roles, ""));
     }
 
     @Test
     void hasSolicitorRoleReturnsTrueWhenRoleEndsWithSolicitor() {
         roles.add(CASEWORKER_BEFTA_JURISDICTION_SOLICITOR);
-        assertTrue(securityUtils.hasSolicitorRole(roles));
+        assertTrue(securityUtils.hasSolicitorRoleForJurisdiction(roles, JURISDICTION));
     }
 
     @Test
     void hasSolicitorRoleReturnsTrueWithMixedCase() {
         roles.add("caseworker-ia-SoliciTor");
-        assertTrue(securityUtils.hasSolicitorRole(roles));
+        assertTrue(securityUtils.hasSolicitorRoleForJurisdiction(roles, "ia"));
     }
 
     @Test
     void hasSolicitorRoleReturnsFalseWithInvalidSuffix() {
         roles.add("caseworker-befta_jurisdiction-barrister");
-        assertFalse(securityUtils.hasSolicitorRole(roles));
+        assertFalse(securityUtils.hasSolicitorRoleForJurisdiction(roles, JURISDICTION));
     }
 
     @Test
     void hasSolicitorRoleReturnsFalseWithAdditionalSuffixAppended() {
         roles.add("caseworker-befta_jurisdiction-solicitorsurname-solicitor-role");
-        assertFalse(securityUtils.hasSolicitorRole(roles));
+        assertFalse(securityUtils.hasSolicitorRoleForJurisdiction(roles, JURISDICTION));
```

Comment:
**[As per #223]** Should there at least be a jurisdiction negative test now that jurisdictions have been re-added to the function? Also may be worth adding a positive test using the failing format `caseworker-ia-legalrep-solicitor` that created ACA-119.